scss/base/_variables.scss

```diff
@@ -28,6 +28,17 @@ $crm-grayblue-darker:       #c2cfd8 !default;
 $crm-white:                 #fff !default;
 $crm-black:                 #000 !default;
 
+//== Typography
+//
+//## Font, line-height, and color for body text, headings, and more.
+
+$crm-font-weight-h1:          600 !default;
```

Comment:
I dont see where these are used. Could you point me please.

Reply:
`$crm-font-weight-h1` specifically has not been used, but `$crm-font-weight-h3` has been used here:
https://github.com/compucorp/uk.co.compucorp.civicase/pull/38/files#diff-8dc7c67508be66b6b72a807f1c27c719R143
```scss
  tbody .btn {
    font-weight: $crm-font-weight-h3;
    padding: 10px 0;
    position: relative;
    width: 100%;
  }
```

I was trying to provide these default values from Shoreditch.

Reply:
`$crm-font-weight-h1` is not being used, but `$crm-font-weight-h2`. These values exist in Shoreditch so I was including them.